src/main/minio.js

```diff
@@ -1421,6 +1421,61 @@ export class Client {
     this.makeRequest({method, bucketName, objectName}, '', 204, '', false, cb)
   }
 
+  // Remove all the objects residing in the objectsList.
+  //
+  // __Arguments__
+  // * `bucketName` _string_: name of the bucket
+  // * `objectsCh` _string_: channel of objects
+  removeObjects(bucketName, objectsList, cb) {
+    if (!isValidBucketName(bucketName)) {
+      throw new errors.InvalidBucketNameError('Invalid bucket name: ' + bucketName)
+    }
+    if (typeof objectsList !== "object") {
```

Comment:
Can this be more specifically checked for an Arry/List?

Reply:
@kannappanr  Done.